pkg/variablefinders/requirements.go

```diff
@@ -95,7 +95,11 @@ func GetSettings(g gitclient.Interface, jxClient jxc.Interface, ns, dir, owner,
 			return nil, "", errors.New("failed to find a dev environment source url on development environment resource")
 		}
 	}
-	clusterDir, err := requirements.PartialCloneClusterRepo(g, gitURL, true, sourceconfigs.SourceConfigFile)
+	combinedConfigFiles := []string{"/" + jxcore.RequirementsConfigFileName, sourceconfigs.SourceConfigFile}
+	if len(combinedConfigFiles) == 0 {
+		return nil, "", errors.New("combined config files are empty")
+	}
```

Comment:
This is redundant as it's always populated

Reply:
Done.